File: Lógica de Programação Java e C#/4 - Controle de Fluxo/C#/Exercícios Complementares/ContaUns.cs
class ContaUns
{
    static void Main()
    {
        System.Random gerador = new System.Random();
        double valor1 = gerador.NextDouble();
        System.Console.WriteLine(valor1);
        string numero = "" + valor1;
        int resposta = 0;
        for (int i = 0; i < numero.Length; i++)
        {
            if (numero[i] == '1')
            {
                resposta++;
            }
        }
        System.Console.WriteLine("Número de Ocorrências " + resposta);
    }
}

File: Lógica de Programação Java e C#/4 - Controle de Fluxo/Java/Exercícios Complementares/ContaUns.java
class ContaUns {
	public static void main ( String [] args ) {
		double numero = Math . random ();
		System.out.println(numero);
		String valor = "" + numero ;
 		int resposta = 0;
		 for ( int i = 0; i < valor.length(); i++) {
 			if(valor.charAt(i)=='1'){
 				resposta ++;
 			}
 		}
		 System . out. println (resposta);
 	}
 }